From 653e229d96d6b3ade74f615044d8d93fe2d058af Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Tue, 3 Apr 2012 16:01:23 -0500
Subject: ENGR00178915: imx6 clock fix build warnings

* Fix build warnings
* clock.c: In function '_clk_pll1_enable':
  warning: no return statement in function returning non-void
* clock.c: In function 'mx6_clocks_init':
  warning: unused variable 'reg'

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index c84d43a..c5074d3 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -531,6 +531,7 @@ static int _clk_pll1_enable(struct clk *clk)
 {
 	_clk_pll_enable(clk);
 	pll1_enabled = true;
+	return 0;
 }
 
 static struct clk pll1_sys_main_clk = {
@@ -5147,7 +5148,6 @@ int __init mx6_clocks_init(unsigned long ckil, unsigned long osc,
 	unsigned long ckih1, unsigned long ckih2)
 {
 	__iomem void *base;
-	unsigned int reg;
 	int i;
 
 	external_low_reference = ckil;
-- 
1.7.9.5

